skills/para-impl/references/star-parallel.md

```
@@ -119,17 +111,17 @@ Workflow の返却は `{conflicts, tickets: [...]}` の形。`conflicts` は衝
 
 ## Phase 11: Worktreeクリーンアップ（複数Issueの場合）
 
-全PRのレビュー対応が完了した後、またはユーザーが明示的にクリーンアップを指示した場合に、各チケットの worktree を削除する。
+全PRのレビュー対応が完了した後、またはユーザーが明示的にクリーンアップを指示した場合に、各 worker の worktree を削除する。
 
 > **前提条件**: 全PRについて以下がすべて完了していること:
 > - セルフレビューの指摘修正が完了
 > - PRレビュー指摘の修正が完了（指摘なしの場合は不要）
 > - 必要な修正のコミット・プッシュが完了
 
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)